src/OSFramework/DataGrid/Feature/ExposedFeatures.ts

```diff
@@ -28,6 +28,7 @@ namespace OSFramework.DataGrid.Feature {
         public sort: IColumnSort;
         public styling: IStyling;
         public tabNavigation: ITabNavigation;
+        public toolTip: ITooltip;
```

Comment:
"tooltip" is a single word, so the property name should reflect that: `tooltip` instead of `toolTip`.

Reply:
Done!